taxi/views.py

```diff
@@ -34,6 +35,26 @@
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = ["name", "country"]
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    template_name = "taxi/manufacturer_list_form.html"
+
+
+class ManufacturerUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Manufacturer
+    fields = ["name", "country"]
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    template_name = "taxi/manufacturer_list_form.html"
```

Comment:
The template name `taxi/manufacturer_list_form.html` used in `ManufacturerUpdateView` might be misleading. Consider renaming it to something more descriptive like `taxi/manufacturer_form.html` to better reflect its purpose.